src/schema/computeResources.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Compute Resources Configuration",
+  "properties": {
+    "cloud-pipelines.net/launchers/generic/resources.cpu": {
+      "type": "number",
+      "title": "CPU",
+      "description": "Number of CPU cores",
+      "minimum": 0,
+      "x-unit": "cores"
+    },
+    "cloud-pipelines.net/launchers/generic/resources.memory": {
+      "type": "number",
+      "title": "Memory",
+      "description": "Memory allocation",
+      "pattern": "^[0-9]+(Gi|Mi|Ki)$",
+      "x-unit": "GiB",
+      "x-append": "Gi",
+      "x-min": 0,
+      "x-max": 2880
+    },
+    "cloud-pipelines.net/launchers/generic/resources.accelerators": {
+      "type": "string",
+      "title": "GPU",
+      "description": "GPU type and quantity",
+      "x-enable-quantity": true,
+      "x-min": 0,
+      "enum": ["NVIDIA-H200"],
+      "x-enum-labels": {
+        "NVIDIA-H200": "NVIDIA H200"
+      }
+    },
+    "cloud-pipelines.net/orchestration/cloud_provider": {
+      "type": "string",
+      "title": "Cloud provider",
+      "enum": ["google", "nebius"],
+      "x-enum-labels": {
+        "google": "Google Cloud",
+        "nebius": "Nebius Cloud"
```

Comment:
I wanted something different here - a separate schema for each cloud provider. Basically a map:
```json
{
    "launcher_annotation_schemas": {
        "google": {...Json Schema 1...},
        "nebius": {...Json Schema 2...},
    }
}
```

Why? Example: GCP and Nebius have different CPU, Memory GPU limits and different GPUs. Actually GCP does not support any GPUs anymore, so we want to hide them completely when Google Cloud is selected.

So, `cloud_provider` essentially acts like a selector - choosing between different Resource Configuration panels.